pom.xml

```diff
@@ -31,21 +31,19 @@
     </developers>
 
     <properties>
-        <java.level>7</java.level>
-        <jenkins.version>1.625.3</jenkins.version>
+        <java.level>8</java.level>
+        <jenkins.version>2.60.3</jenkins.version>
```

Comment:
2.176 is the latest LTS (and even that is probably superseded by 2.190 now).  Any objection to going to that instead?  Assuming the next big issue to solve is pipeline compatibility, probably good to stay as recent as possible...

Reply:
Many plugins use 2.60.x as the minimum to provide Java 8. This also makes sure that more users can install the plugin. I am not sure if users should be forced to update Jenkins itself via plugin updates. I think the Jenkins version should state minimum required one that contains the required functionality for a plugin.

Anyway, if you want me to update to 2.176.x let me know and I'll do so.